app/views/shared/_favicons.html.haml

```diff
@@ -0,0 +1,14 @@
+- if Settings.favicon_paths.x16.present?
```

Comment:
Here's what we did for dartmouth: https://github.com/tablexi/nucore-dartmouth/blob/master/vendor/engines/dartmouth_radar/lib/dartmouth_radar/engine.rb and the associated icon file: https://github.com/tablexi/nucore-dartmouth/blob/master/vendor/engines/dartmouth_radar/app/views/dartmouth_radar/_icons.html.erb

We have a lot more icons including for windows. If we do want to bake this in, I wonder if there's a way to make the settings a little more structured and also allow more flexibility.

```yaml
favicons:
  - path: radar/icons/apple-touch-icon-57x57.png
    sizes: 57x57
    rel: apple-touch-icon-precomposed
  - ...
  - path: radar/icons/favicon-196x196.png
    sizes: 196x196
    rel: icon
```

Reply:
If you think it's a good idea, maybe I could write something that loads the directory and automatically renders them based on file name.

Reply:
That's not a bad idea. If it's something that can be done at application launch and cached then I think it's a good idea. I wouldn't want to need to do a disk read on every single request. We haven't really done anything with caching in this application, so I don't believe we have Rails's caching set up.

Reply:
I'm going to try myself a bit, but that sounds beneficial.